README.md

```diff
@@ -1466,6 +1467,24 @@ Currently being reworked as Neo Launcher.
 - [x] [Github](https://github.com/Crazy-Marvin/VacationDays)
 - [x] [Official page](https://crazymarvin.com/vacation-days/)
 
+## Translator
+
+### Arna Translate
+<img alt="SimplyTranslate Mobile" height="64" src="https://user-images.githubusercontent.com/16052180/171461601-8b58706b-12d4-4131-97ad-1580fd5ba580.png">
+
+- [ ] Google Play
+- [x] [IzzyOnDroid](https://apt.izzysoft.de/fdroid/index/apk/com.arna.translate)
+- [x] [GitHub](https://github.com/MahanRahmati/translate)
+- [x] [Official page](https://lingva.ml/)
+
+### SimplyTranslate Mobile ❤️
+<img alt="SimplyTranslate Mobile" height="64" src="https://github.com/ManeraKai/simplytranslate_mobile/raw/main/fastlane/metadata/android/en-US/images/icon.png">
+
+- [ ] Google Play
+- [x] [F-Droid](https://f-droid.org/en/packages/com.simplytranslate_mobile/)
+- [x] [GitHub](https://github.com/ManeraKai/simplytranslate_mobile)
+- [x] [Official page](https://simplytranslate.org/)
```

Comment:
Should not be this site? https://manerakai.github.io/simplytranslate_mobile/

Reply:
yes i choose to use the instance instead, should i remove it. also did that for arna translate